composer.json

```diff
@@ -7,9 +7,9 @@
     "require-dev": {
```

Comment:
Ok! I value your experience with the libs in the PHP ecosystem. I'm impressed that devs are consistent enough with semver that we can consider leaving a patch version unspecified.